src/client/src/components/SelectableCard/index.tsx

```diff
@@ -114,14 +118,24 @@ const SelectableCard = ({
           />
         </Link>
         <div className={styles.pageButtons}>
+          {isPagesSelection && (
+            <button
+              className={classNames(styles.cardCount, styles.countButton)}
+              onClick={() => {
+                removePage(cardNumber);
```

Comment:
nit: I think you can make this into one line, i.e.
`() => removePage(cardNumber)`